### ISSUE TITLE
Paket copy "local" pacakges to cache

### ISSUE THREAD

**Issue by lexarchik** [fsprojects/Paket#2703]
Paket copies packages overrided in paket.local file to "global" cache (`~/.nuget`) during restore process.

[Documentation](https://fsprojects.github.io/Paket/local-file.html) says "The `paket.local` file is used for testing changes before releasing new version of a dependent project."

Paket copies to cache "experimental" version of package and poor nuget manager can accidentally take it:)

**Comment by forki** [comment]
What is the issue? And what is the suggested solution?

**Comment by matthid** [comment]
This is related to the two "File aleady exists" reports. The issue is that we probably should  skip the cache for these packages, otherwise behavior will be hard to follow for users.
Especially in combination with nuget which will pick the wrong version (but another paket from another repository will do it as well)

**Comment by lexarchik** [comment]
Actually i can't find a way in which paket or nuget manager restore "local" version of package from cache.

But i use `~/.nuget` as package source sometimes and "local" package breaks my workflow :)
![workflow](https://imgs.xkcd.com/comics/workflow.png)!

And saving "local" packages to cache seems conceptually wrong.

**Comment by matthid** [comment]
probably duplicate of https://github.com/fsprojects/Paket/issues/2690
(at least we will solve it as part of this, as we shouldn't use the cache at all when paket.override is used)
